projects/js-packages/charts/.gitattributes

```
@@ -10,4 +10,5 @@ node_modules      export-ignore
 /changelog/**     production-exclude
 /jest.config.cjs   production-exclude
 **/stories/**     production-exclude
-**/test/**        production-exclude
+**/tests/**       production-exclude
+**/src/**	      production-exclude
```

We can exclude the newly added `docs` dir too
```suggestion
**/src/**         production-exclude
**/docs/**        production-exclude
```